src/openrct2/ride/VehiclePaint.cpp

```diff
@@ -1323,8 +1337,9 @@ static void vehicle_sprite_0_14(
     }
     if (vehicleEntry->sprite_flags & VEHICLE_SPRITE_FLAG_INLINE_TWISTS)
     {
-        int32_t boundingBoxNum = ((imageDirection / 8) ^ 2) + 140;
-        int32_t spriteNum = (((imageDirection / 8) + 36) * vehicleEntry->base_num_frames) + vehicleEntry->inline_twist_image_id;
+        int32_t boundingBoxNum = (YawTo4(imageDirection) ^ 2) + 140;
```

Comment:
I'm sure you already know this but a ^2 on a yaw4 is reversing the yaw i.e. 0,1,2,3 {N,E,S,W} becomes 2,3,0,1 {S,W,N,E} likewise a ^4, ^8, ... on a yaw8, yaw16, ... it would make sense to make a function for this (perhaps in a future PR).